client/signup/jetpack-connect/sso.jsx

```diff
@@ -1,14 +1,16 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
 import debugModule from 'debug';
 import get from 'lodash/get';
 import map from 'lodash/map';
 import Gridicon from 'gridicons';
 import cookie from 'cookie';
+import flowRight from 'lodash/flowRight';
```

Comment:
Maybe move all lodash imports together, like this:

`import { flowRight, get, map } from 'lodash';`